core/templates/pages/exploration-editor-page/editor-tab/state-name-editor/state-name-editor.component.html

```
@@ -12,7 +12,7 @@
 
       <span *ngIf="stateNameService.isStateNameEditorShown()">
         <form class="row mb-3" role="form" #f="ngForm" (ngSubmit)="saveStateNameAndRefresh(tmpStateName)">
-          <input type="text" name="tmpStateName" [(ngModel)]="tmpStateName" [maxlength]="maxLen" (focus)="stateNameEditorOpened" autofocus class="e2e-test-state-name-input">
+          <input type="text" name="tmpStateName" [(ngModel)]="tmpStateName" [maxlength]="maxLen" autofocus class="e2e-test-state-name-input">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)